manifests/i/IBM/SemeruRuntimeOpenEdition/JDK/17/17.0.9+9/IBM.SemeruRuntimeOpenEdition.JDK.17.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: IBM.SemeruRuntimeOpenEdition.JDK.17
+PackageVersion: 17.0.9+9

```

```suggestion
PackageVersion: 17.0.9.9
```
Should match control panel exactly. Version folder will need to be renamed after making this change